package/MDAnalysis/coordinates/timestep.pxd

```diff
@@ -0,0 +1,31 @@
+from libcpp cimport bool
+from libc.stdint cimport int64_t, uint64_t
+cimport numpy as cnp
+cnp.import_array()
+
+
+cdef class Timestep:
+
+    cdef uint64_t _n_atoms
+    cdef public int64_t  frame
+
+
+    # info for numpy C API
+    cdef int _typenum
+    cdef cnp.npy_intp _particle_dependent_dim[2]
```

Comment:
is the second dimension always 3?  &if so isn't this variable always `_n_atoms, 3`?

Reply:
Yeah its set to `_n_atoms, 3` on `__cinit__`